packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -223,6 +228,28 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
         Debug.inDevelopmentMode(() => deepFreeze(this));
     }
 
+    private findSeriesWithUserVisiblity(newUserOptions: T, deltaOptions: DeepPartial<T> | null | undefined) {
+        for (const o of [newUserOptions, deltaOptions]) {
+            const series = o?.series;
+            if (Array.isArray(series)) {
```

Comment:
Nit: Make this a short-circuit in the negative case to reduce nesting here?
```suggestion
            if (!Array.isArray(series)) continue;
```